fastestimator/util/latex_util.py

```diff
@@ -129,3 +129,54 @@ def __init__(self,
             data.append(bold(escape_latex(name)) if bold_name else escape_latex(name))
         data.append(NoEscape("}}}"))
         super().__init__(data=data)
+
+
+class TabularCell(Container):
+    """A class to convert iterable to latex representation.
+
+    The data of this class can be any type. Usually it is iterable due to its capability to setup interval string.
+
+    Args:
+        data: Data of the cell.
+        token: String to be added in the interval of data entries.
+    """
+    def __init__(self, data: list, token: str):
+        super().__init__(data=data)
+        self.token = token
+
+    def dumps(self) -> str:
+        """Get a string representation of this cell.
+
+        Returns:
+            A string representation of itself.
+        """
+        return dumps_list(self, token=self.token)
+
+
+class WrapText(LatexObject):
+    """A class to convert string or number to wrappable latex representation.
+
+    This class will first convert the data to string, and then to wrappable latex representation if its length is too
+    long. This is to fix the issue that first element and number of Tabularx cells is not wrappable in X column type.
+
+    Args:
+        data: String data.
+        seq_thld: When length of string is above this number, the string will be converted to wrappable text by being
```

Comment:
Can we call this 'threshold' instead? Also, typically in the doc strings when we are referencing one of the other arguments we would say "When the length of `data` is greater than `threshold`, the resulting string will be made wrappable" (with the angular quotes)